tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/asynchronous/AsyncClassLevelClient.java

```diff
@@ -52,8 +52,12 @@
     }
 
     /**
-     * Service an operation until waitCondition is completed or 1 second timeout.
-     *
+     * Service an operation until waitCondition is completed or 1000 second timeout.
+     * NOTE: The timeout used in the waiting condition (1000 SECONDS) will simulate a large service processing time, to allow use in the
+     * test cases a strategy to improve the waiting time and avoid has process waiting for ever the strategy is described below:
```

Comment:
I suggest to delete this:
_, to allow use in the test cases a strategy to improve the waiting time and avoid has process waiting for ever the strategy is described below: 
n the cases that are necessary test if the @{@link Future} is finished we can me make this finish immediately and use awaitility library to control at most how much time will wait for the test before to finish in cases that the service takes more time due to external factors like infrastructure in a continuous integration context or stress testing.
_
as it does not read well and does not add much.

Please just add:

This 1000 second timeout is to ensure test timeout kicks in before the operation timeout for a better test error to be displayed.

Reply:
done.